frontend/src/features/properties/selector/MapSelectorContainer.tsx

```diff
@@ -71,7 +71,7 @@ export const MapSelectorContainer: React.FunctionComponent<IMapSelectorContainer
           Add to selection
         </Button>
       ) : null}
-      <StyledFormSection>
+      <Section header={undefined}>
```

Comment:
left as a marker so that the H3 below can be removed if we decided to have sections with h2 or h3 headings